analysis/study_definition.py

```diff
@@ -1,33 +1,191 @@
+    """
+    definitions={}
+    for (var_name, med_codelist_file) in meds_list:
+        definitions.update(medication_counts_and_dates(var_name, med_codelist_file))
+    return definitions
 
 study = StudyDefinition(
     # Configure the expectations framework
     default_expectations={
         "date": {"earliest": "1900-01-01", "latest": "today"},
-        "rate": "exponential_increase",
+        "rate": "uniform",
+        "incidence": 0.1,
     },
     # This line defines the study population
     population=patients.registered_with_one_practice_between(
-        "2019-02-01", "2020-02-01"
+        "2019-03-01", "2020-03-01"
+    ),
+    # Outcomes
+    icu_date_admitted=patients.admitted_to_icu(
+        on_or_after="2020-03-01",
+        include_day=True,
+        returning="date_admitted",
+        find_first_match_in_period=True,
+        return_expectations={"date": {"earliest": "2020-03-01"}, "incidence": 0.1},
+    ),
+    died_date_cpns=patients.with_death_recorded_in_cpns(
+        on_or_before="2020-06-01",
+        returning="date_of_death",
+        include_month=True,
+        include_day=True,
+        return_expectations={"date": {"earliest": "2020-03-01"}},
+    ),
+    died_ons_covid_flag_any=patients.with_these_codes_on_death_certificate(
+        covid_identification, on_or_after="2020-03-01", match_only_underlying_cause=False,
+        return_expectations={"date": {"earliest": "2020-03-01"}, "incidence": 0.1},
+    ),
+    died_ons_covid_flag_underlying=patients.with_these_codes_on_death_certificate(
+        covid_identification, on_or_after="2020-03-01", match_only_underlying_cause=True,
+        return_expectations={"date": {"earliest": "2020-03-01"}, "incidence": 0.1},
+    ),
+    died_date_ons=patients.died_from_any_cause(
+        on_or_after="2020-06-01",
+        returning="date_of_death",
+        include_month=True,
+        include_day=True,
+        return_expectations={"date": {"earliest": "2020-03-01"}, "incidence": 0.1},
+    ),
+    # COVID-19 outcomes
+    first_pos_test_sgss=patients.with_test_result_in_sgss(
+        pathogen="SARS-CoV-2",
+        test_result="positive",
+        find_first_match_in_period=True,
+        returning="date",
+        date_format="YYYY-MM-DD",
+        return_expectations={"date": {"earliest": "2020-01-01"}},
+    ),
+
+    first_pos_code_primcare=patients.with_these_clinical_events(
+        covid_pos_primcare_code,
+        returning="date",
+        find_first_match_in_period=True,
+        date_format="YYYY-MM-DD",
+        return_expectations={
+            "date": {"earliest": "2020-01-01", "latest": "today"}
+        },
+    ),
+
+    first_pos_test_primcare=patients.with_these_clinical_events(
+        covid_pos_primcare_test,
+        returning="date",
+        find_first_match_in_period=True,
+        date_format="YYYY-MM-DD",
+        return_expectations={
+            "date": {"earliest": "2020-01-01", "latest": "today"}
```

Comment:
we discuss using SGSS for a sensitivity analysis but not primary care.

Reply:
Primary care removed.